lib/core/src/Cardano/Wallet/Primitive/CoinSelection/MA/RoundRobin.hs

```diff
@@ -411,10 +408,12 @@ performSelection
         -- ^ A function that computes the extra cost corresponding to a given
         -- selection. This function must not depend on the magnitudes of
         -- individual asset quantities held within each change output.
+    -> (TokenBundle -> TokenBundleSizeAssessment)
+        -- ^ A function that assesses the size of a token bundle.
```

Comment:
As we said in the call: we should describe the properties we expect from this function.

Reply:
Hi @Anviking 

I've added some documentation in commit 1d8f422f54aee089260c0a53c5d1fe287a8d9af7.

I'm also going to add some property tests for the CBOR implementation of `TokenBundleSizeAssessor`.

Reply:
Property tests added in commit 7f0059034d661b997933f1dc043f734d36a26c4e.